examples/getting-started-movielens/03-Training-with-PyTorch.ipynb

```diff
@@ -37,52 +37,44 @@
     "\n",
```

Comment:
<p>is it really the length or more like starting point?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>made the change</p>